### ISSUE TITLE
Support @inheritDoc without curly braces

### ISSUE THREAD

**Issue by Stadly** [phpstan/phpstan-strict-rules#62]
For the following code, PHPStan will issue an error: `Return type (Traversable) of method Bar1::convert() should be covariant with return type (iterable<string>&Traversable) of method Foo::convert()`.

The most common way to inherit PHPDoc explicitly is to use `{@inheritDoc}` (as in `Bar2`), but using `@inheritDoc` (as in `Bar1`) is the correct way, according to the [spec draft](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag).

```php
interface Foo
{
    /**
     * @return Traversable<string>
     */
    public function convert(): Traversable;
}

abstract class Bar1 implements Foo
{
    /**
     * @inheritDoc
     */
    abstract public function convert(): Traversable;
}

abstract class Bar2 implements Foo
{
    /**
     * {@inheritDoc}
     */
    abstract public function convert(): Traversable;
}

abstract class Bar3 implements Foo
{
    abstract public function convert(): Traversable;
}
```

**Comment by Stadly** [comment]
Yes. Thanks for reopening the other issue :)
